CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## [Upcoming changes][Unreleased]
 
+## [2.2.7] - 2017-11-21
+
+### Fixed
+
+* enable touch selection for results #186
+* formatted display text bug d5de54b12f9b17e11c46db8d6cea767190420a2e
+
```

Comment:
i don't know _why_, but issues/prs and commit hashes are **not** hyperlinked automatically in markdown files

```md
[#186](https://github.com/Esri/esri-leaflet-geocoder/issues/186)
[`d5de54b`](https://github.com/Esri/esri-leaflet-geocoder/commit/d5de54b12f9b17e11c46db8d6cea767190420a2e)
```

Reply:
thanks! should now be fixed.